Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM golang:1.19
+
+WORKDIR /usr/src/app
+
+COPY go.mod go.sum ./
+RUN go mod download && go mod verify
+
+COPY . .
+
+RUN make
+
+CMD ["bin/tadpoles-backup", "--help"]
```

Comment:
I'd like to minimize the size of the image by using build stages. Also no need to nest the paths.
```suggestion
FROM golang:1.19 AS builder

WORKDIR /code

# this is broken out from call to `make` to improve docker caching
COPY go.mod go.sum ./
RUN go mod download && go mod verify

# run main build
COPY . .
RUN  make dev

FROM builder

WORKDIR /app
COPY --from=builder /code/bin/tadpoles-backup .

ENTRYPOINT ["./tadpoles-backup"]

CMD ["version"]
```
setting `ENTRYPOINT` will remove the redundancy of specifying the executable each time:
`docker run -it --rm -v<volumes> tadpoles backup data`
the image name becomes like the command itself.